challenges/workflow-engine.md

```diff
@@ -1,8 +1,13 @@
 # Workflow Engine
 
-**Allotted Time:** 1 hour
+This assignment utilizes the Sophos Factory platform. You'll build one or more [step modules](https://docs.refactr.it/docs/reference/step-module-reference/) that execute a script in the language of your choice to solve the challenge.
 
-The goal of this challenge is to implement a workflow engine that supports simple templating. Given the below JSON object, write a program that executes the workflow.
+The goal of this challenge is to implement a workflow engine that supports simple templating. Given the below JSON object, write two script modules:
+
+1. A module that parses these instructions, and
+2. A module that executes the workflow.
+
+Create a [pipeline](https://docs.refactr.it/docs/building-pipelines/) that takes the below JSON as input, executes the instructions using your two modules above, and properly logs output to the run logs.
```

Comment:
Might be good to emphasize that the candidate is not expected to create an application that provides a pipeline creation feature. Also not to use the app, since there could maybe be confusion thinking that the SPA must be used, thus spending time trying to find out how to sign up in order to use the SPA.